README.md

```diff
@@ -69,6 +68,25 @@ This runs 3 processes for both apps and the backend services on 3 different port
 - 3001 for the partners app
 - 3100 for backend/core
 
+### Bloom's UI-Component Development
+- Although its own open-source repository, developers may wish to see ui-component changes in the Bloom context. This is achieved with the following steps:
```

Comment:
Just a grammar issue here, would suggest `Because Bloom's ui-components package is a separate open source repository, developing within both repos locally requires linking the folders with the following steps:`

Reply:
I prefer yours as well, thank you!